packages/ketcher-react/src/script/ui/component/form/MeasureInput/measure-input.tsx

```diff
@@ -92,8 +92,12 @@ function convertValue(value, measureFrom, measureTo) {
   if ((!value && value !== 0) || isNaN(value)) return null // eslint-disable-line
 
   return measureTo === 'px' || measureTo === 'pt'
-    ? ((value * measureMap[measureFrom]) / measureMap[measureTo]).toFixed() - 0
-    : ((value * measureMap[measureFrom]) / measureMap[measureTo]).toFixed(3) - 0
+    ? Number(
+        ((value * measureMap[measureFrom]) / measureMap[measureTo]).toFixed()
+      ) - 0
+    : Number(
+        ((value * measureMap[measureFrom]) / measureMap[measureTo]).toFixed(3)
+      ) - 0
```

Comment:
Looks like these two lines only differ in that the second one uses "toFixed(3)" - maybe we could put this into a separate function? e.g. `getFormatedValue` or something like that?

Reply:
Done